### ISSUE TITLE
Update version constraints

### ISSUE THREAD

**Issue by begriffs** [dan-t/hsimport#9]
Hello, hsimport looks super helpful! I had trouble installing it with Stack however. Ultimately I was able to do so by removing upper bounds on package versions in the cabal file, but I realize that's pretty drastic. Could you adjust the upper bounds? Here are the packages which `packdeps` says have moved ahead of your upper bounds.

```sh
$ packdeps hsimport.cabal
hsimport-0.6.5: Cannot accept the following packages
attoparsec 0.13.0.1
lens 4.13
tasty 0.11
tasty-golden 2.3.0.1
```

**Comment by dan-t** [comment]
I just released hsimport 0.6.6 to fix it.

**Comment by begriffs** [comment]
So fast! Thank you. :watch: :heart:
